xml/vk.xml

```diff
@@ -7563,6 +7569,14 @@ typedef void <name>CAMetalLayer</name>;
             <param><type>VkPerformanceParameterTypeINTEL</type> <name>parameter</name></param>
             <param><type>VkPerformanceValueINTEL</type>* <name>pValue</name></param>
         </command>
+        <command successcodes="VK_SUCCESS" errorcodes="VK_ERROR_INITIALIZATION_FAILED">
+            <proto><type>VkResult</type> <name>vkAcquireWaylandDisplayEXT</name></proto>
+            <param><type>VkPhysicalDevice</type> <name>physicalDevice</name></param>
+            <param>struct <type>wl_display</type>* <name>display</name></param>
+            <param>struct <type>zwp_drm_lease_manager_v1</type>* <name>manager</name></param>
+            <param><type>int</type> <name>nConnectors</name></param>
+            <param><type>VkWaylandLeaseConnectorEXT</type>* <name>pConnectors</name></param>
```

Comment:
We use `uint32_t` type for array length.
Also count variable would be named like `connectorCount`.
Also missing `len="queueCount"`.